spec/Section 2 -- Language.md

```diff
@@ -516,6 +516,74 @@ which returns the result:
 }
 ```
 
+## Nullability
+
+Nullability :
+  - !
+  - ?
+
+Fields can have their nullability designated with either a `!` to indicate that a
+field should be `Non-Nullable` or a `?` to indicate that a field should be 
+`Nullable`. These designators override the nullability set on a field by the schema 
+for the operation where they're being used. For example, a field marked with `!` in 
+a query will be treated identically to a field marked with `!` in the schema for the
+purposes of validation and execution.
```

Comment:
@leebyron I didn't make any changes to the Execution document because I was thinking this would cover it, but I can specify there too if you think that would help.

Reply:
I'll have to read carefully, but I'm pretty sure we extract the type from the schema and do error propagation rules accordingly. At a minimum we should have a note about the impact of nullability qualifiers so it's not just implicit, but may also need explicit steps in those algorithms.